scripts/rollup/config.js

```diff
@@ -5,6 +5,12 @@ import replace from 'rollup-plugin-replace';
 import { uglify } from 'rollup-plugin-uglify';
 import filesize from 'rollup-plugin-filesize';
 
+if (process.env.NODE_ENV === 'production' && !process.env.VERSION) {
```

Comment:
Since we're at it, we should even check that `process.env.VERSION` follows semver to avoid any surprises. We already use the `semver` package in the codebase. Wdyt?

Reply:
that's a good idea!

Reply:
On it !